oxd-gen-client/src/test/resources/testng.xml

```diff
@@ -0,0 +1,131 @@
+    <parameter name="licenseServerEndpoint" value="https://${oxd.server.address}/license-server"/>
+    <parameter name="rsProtect"
+               value="{'resources':[{'path':'/ws/phone','conditions':[{'httpMethods':['GET'],'scopes':['http://photoz.example.com/dev/actions/all','http://photoz.example.com/dev/actions/view'],'ticketScopes':['http://photoz.example.com/dev/actions/view']},{'httpMethods':['PUT', 'POST'],'scopes':['http://photoz.example.com/dev/actions/all','http://photoz.example.com/dev/actions/add'],'ticketScopes':['http://photoz.example.com/dev/actions/add']},{'httpMethods':['DELETE'],'scopes':['http://photoz.example.com/dev/actions/all','http://photoz.example.com/dev/actions/remove'],'ticketScopes':['http://photoz.example.com/dev/actions/remove']}]}]}"/>
+    <parameter name="rsProtectScopeExpression"
+               value="{'resources':[{'path':'/ws/phone','conditions':[{'httpMethods':['GET'],'scopes':['http://photoz.example.com/dev/actions/all','http://photoz.example.com/dev/actions/view'],'scope_expression':{'rule':{'and':[{'or':[{'var':0},{'var':1}]},{'var':2}]},'data':['http://photoz.example.com/dev/actions/all','http://photoz.example.com/dev/actions/add','http://photoz.example.com/dev/actions/view']}},{'httpMethods':['PUT','POST'],'scope_expression':{'rule':{'and':[{'or':[{'var':0},{'var':1}]},{'var':2}]},'data':['http://photoz.example.com/dev/actions/all','http://photoz.example.com/dev/actions/add','http://photoz.example.com/dev/actions/view']},'ticketScopes':['http://photoz.example.com/dev/actions/add']},{'httpMethods':['DELETE'],'scopes':['http://photoz.example.com/dev/actions/all','http://photoz.example.com/dev/actions/remove'],'ticketScopes':['http://photoz.example.com/dev/actions/remove']}]}]}"/>
+    <parameter name="rsProtectScopeExpressionSecond"
+               value="{'resources':[{'path':'/GetAll','conditions':[{'httpMethods':['GET'],'scopes':['http://photoz.example.com/dev/actions/a1','http://photoz.example.com/dev/actions/a2','http://photoz.example.com/dev/actions/a3'],'ticketScopes':['http://photoz.example.com/dev/actions/a1','http://photoz.example.com/dev/actions/a2','http://photoz.example.com/dev/actions/a3'],'scope_expression':{'rule':{'and':[{'or':[{'var':0},{'var':1}]},{'var':2}]},'data':['http://photoz.example.com/dev/actions/a1','http://photoz.example.com/dev/actions/a2','http://photoz.example.com/dev/actions/a3']}}]}]}"/>
+
+    <!-- main class to set up and tear down suite -->
+   <test name="SetUp" enabled="true">
+        <classes>
+            <class name="io.swagger.client.api.SetUpTest"/>
+        </classes>
+    </test>
+
+    <!-- Connect -->
+    <test name="Register site" enabled="true">
+        <classes>
+            <class name="io.swagger.client.api.RegisterSiteTest"/>
+        </classes>
+    </test>
+<!--
+    <test name="Remove site" enabled="true">
+        <classes>
+            <class name="io.swagger.client.api.RemoveSiteTest"/>
```

Comment:
I've commented out the tests that are not modified according to swagger client. So, this test suit runs with Maven, and from Idea. Not tried on Jenkins.

Reply:
Good thanks. With `<suite-file path="suite1.xml" />` it will be run automatically.

Reply:
Please don't merge now. I think I've to modify this file a bit to make it behave like existing test files. Earlier I had designed it with the mind that it would be run independently. I'll commit the updated file(s) ASAP.

Reply:
Sure, I've rejected this pull request. Once you correct everything just put it in review again and I will review. If all is good I will merge it.